ocs_sample_library_preview/Streams.py

```diff
@@ -21,159 +19,14 @@ def __init__(self, client: BaseClient):
         """
         :param client: base client that handles auth and base routing
         """
+        super().__init__(client=client, collection='Streams')
```

Comment:
Good move separating stream views into a new client, not sure why they were in this one.

Reply:
Thanks!